interface-definitions/service_ipoe-server.xml.in

```diff
@@ -112,6 +112,22 @@
             </children>
           </tagNode>
           #include <include/name-server-ipv4-ipv6.xml.i>
+          <node name="client-ip-pool">
+            <properties>
+              <help>Client IP pools and gateway setting</help>
+            </properties>
+            <children>
+              <tagNode name="name">
+                <properties>
```

Comment:
Is there no constraint on the pool name? I would suggest to add a constraint to not make Accel-PPP fail.

Reply:
@sever-sever can you make changes accordingly?